src/test/starknet.t.sol

```diff
@@ -105,7 +115,25 @@ contract StarknetTests is DssSpellTestBase, ConfigStarknet {
         _checkStarknetDaiBridge();
         _checkStarknetGovRelay();
         _checkStarknetCore();
-        _checkStarknetMessage(l2Spell);
+        _checkStarknetMessage(starknetValues.l2_spell);
+    }
+
+    function testStarknetEventEmission() public {
+        // Ensure the Pause Proxy has some ETH for the Starknet Spell
+        assertGt(pauseProxy.balance, 0);
+        setValues();
```

Comment:
These 2 lines should probably be inside the if clause below.

Reply:
Need to set the values for the if clause to work.

Reply:
Refactored the setValues into test constructor so it only needs to be called once. (setUp is not available because this is inherited from spell base)

https://github.com/makerdao/spells-goerli/pull/165/commits/cfdad4063d1fc2d40ff1aa242b46081e27816f41